participants/Sumana18/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,11 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(number):
```

Comment:
Please rename this pull request to reflect the addition of a new function. I don't think you are fixing a bug here. If you are fixing a bug here, can you please indicate where the bug was fixed?